siteMap/siteMap.yaml

```diff
@@ -177,6 +177,16 @@
         - name: "Contributing Data"
           key: "/contribute/data/contributing-to-hca"
           path: "/contribute"
+          secondaryLinks:
```

Comment:
@NoopDog I updated the siteMap.yaml file (which I think I'm supposed to do) to what I think it should be, but I'm not confident it's right!